### ISSUE TITLE
[Bug]home button behavoiur nighlty 

### ISSUE THREAD

**Issue by ale82to** [mozilla-mobile/fenix#20273]
so after about 2 weeks home button behavoiur has still not been fixed.
why does home button always open new tabs instead of opening top sites /searches in the existing tab it originates from? 
seriously PLEASE FIX IT.

**Comment by kbrosnan** [comment]
@ale82to this is a borderline abusive comment. Removing the information in the current tab is a destructive action. Yes, this might be different behavior than desktop.

**Comment by ale82to** [comment]
every single mobile or desktop browsers has the behavior I described : open top sites or searches in the existing tab so why is fenix behaving so different? 
there is NOTHING DISTRUCTIVE in the behavior I described!! 
there are tons of way to open a new tab : long pressing tab switcher icon, plus icon in tab tray or even long pressing fenix home icon. 
BUT THERE IS NO WAY TO OPEN A TOP SITES IN AN EXISTING TAB!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 
I M quite sure the developers are adductin excuses to disguise the fact That fenix does not have a real home page but a navgraph or something like that. AT LEAST BE HONEST instead of saying my comment borderline abusive.
